### DISCUSSION TITLE
How to filter a sheet to "rows which in a certain Frequency Table would have aggregator with value = X"?

### DISCUSSION THREAD

**Discussion by spencerwilson** [saulpw/visidata#1391] [Q&A] [answered]
I have a sheet with the following data: a string column and a nullable integer column:
```csv
apple,
apple,
banana,2
banana,23
clementine,5
clementine,
dragonfruit,
dragonfruit,
```

My goal is to get a sheet containing rows for which a `count` aggregator on the integer column equals 0. That is, get a sheet with the following:
```csv
apple,
apple,
dragonfruit,
dragonfruit,
```

The closest I know how to get currently is to

1. Add a `count` aggregator to the integer column.
2. Make a Frequency Table on the first column. This gets the dataset
```csv
apple,0
banana,2
clementine,1
dragonfruit,0
```
3. Make a Frequency Table on that ^ sheet's second column.
4. Dive into the "0" row. The result is a subset of ^^ that sheet:
```csv
apple,0
dragonfruit,0
```

I can dive into each of these to see the underlying
```
apple,
apple,
```
and
```
dragonfruit,
dragonfruit,
```
rows. One feature that would solve my problem would be: on a FreqTableSheet, be able to select multiple rows, and then open a sheet containing the union of the selected groups. Is that possible, or is there another path to getting the desired result?

P.S. My heartfelt gratitude to the VisiData maintainers. This tool has enabled me to do analysis that would have been extremely painful otherwise.

**Comment by saulpw** [comment, marked as answer]
If you select rows on a FreqTable, the underlying rows on its source sheet will become selected.  This should chain, so you should be able to select the `0` row on the second FreqSheet, and then quit back to the source sheet and `"` to pull them off into their own sheet.

Or, you could quit back to the first FreqSheet, and press `g Enter` if you want to dive into all selected rows.

**Reply by spencerwilson** [reply]
Indeed, I was missing both of those pieces. Thank you!

> If you select rows on a FreqTable, the underlying rows on its source sheet will become selected.

Moments before you replied I found this info in "How to filter for grouped or described rows", https://www.visidata.org/docs/group/ 🤦‍♂️ 

> `g Enter` to dive into all selected groups

Indeed, both the semantics of `s`/`t`/`u` and of `gEnter` are explained clearly in the Frequency Table section of the man page.

Excellent work ❤️
